libpod/image/image.go

```diff
@@ -307,7 +319,9 @@ func (ir *Runtime) GetImages() ([]*Image, error) {
 		return nil, err
 	}
 	for _, i := range images {
-		newImages = append(newImages, ir.newFromStorage(&i))
+		image := i
```

Comment:
This can go away, you can just &i on the next line

Reply:
nope, this is the bug I was debugging.